src/main/java/uk/gov/digital/ho/hocs/document/JSONToSimpleTextConverter.java

```diff
@@ -17,8 +18,14 @@ public class JSONToSimpleTextConverter {
     private final String inputJson;
     private final StringBuilder convertedOutput = new StringBuilder();
 
-    public JSONToSimpleTextConverter(String inputJson) throws IOException {
+    private final ObjectMapper objectMapper;
+
+    private final EnumMappingsRepository enumMappingsRepository;
+
+    public JSONToSimpleTextConverter(String inputJson, ObjectMapper objectMapper, EnumMappingsRepository enumMappingsRepository) throws IOException {
         this.inputJson = inputJson;
+        this.objectMapper = objectMapper;
```

Comment:
Might be worth getting rid of the ObjectMapper on line 38 if this one exists - otherwise this one isn't used.

Reply:
Hmm for some reason if I remove the ObjectMapper declaration at line 38, the JSONToSimpleTextConverterTest starts failing

Reply:
You will need to either create a new one, or change the test to instantiate the Spring Context and utilise `@SpyBean`. I'd prefer the latter...